enctests/testframework/utils/utils.py

```diff
@@ -123,6 +122,13 @@ def create_media_reference(path, source_clip, is_sequence=False):
                 seq.length(), rate
             )
         )
+
+        #prefix = seq.head()
+        # Lets strip off any periods or "_" at the end of a image sequence
+        # so image.0001.jpg would become image
+        #if prefix[-1] in [".", "_"]:
+        #    prefix = prefix[:-1]
+
```

Comment:
Just curious why this is commented out?

Reply:
This was causing problems in where the test files were, I believe I've now fixed that.

I've removed this code.